src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -55,6 +60,25 @@ public static string GetClassification(SyntaxToken token)
             return null;
         }
 
+        private static bool IsControlKeyword(SyntaxToken token)
+        {
+            if (!SyntaxFacts.IsControlKeyword(token.Kind()))
+            {
+                return false;
+            }
+
+            if (token.IsKind(SyntaxKind.DefaultKeyword))
+            {
+                return token.Parent.IsKind(SyntaxKind.DefaultSwitchLabel);
+            }
+            else if (token.IsKind(SyntaxKind.CaseKeyword))
+            {
+                return token.Parent.IsKind(SyntaxKind.CaseSwitchLabel, SyntaxKind.CasePatternSwitchLabel);
+            }
+
+            return SyntaxFacts.IsControlStatement(token.Parent.Kind());
```

Comment:
that you ask SyntaxFacts, but hten overrule it means this should not be in syntaxfacts.  Because now our LS 'IsControlKeyword' is out of sync with SyntaxFacts.IsControlKeyword.  That's just confusing.  There shouldn't be differences between IsControlKeyword(token) and IsControlKeyword(token.Kind());